packages/postgres/src/sql-template.ts

```diff
@@ -1,31 +1,31 @@
+import { type CompiledQuery, init, PostgresDialect } from '@sejohnson/tql';
 import { VercelPostgresError } from './error';
 
-export type Primitive = string | number | boolean | undefined | null;
```

Comment:
the type of `values` is now `unknown[]` instead of `Primitive[]`, but this is backwards-compatible because all `Primitive` values are assignable to `unknown`.